core/src/luigi-config/auth.js

```diff
@@ -30,8 +30,7 @@ export const createAuth = async () => {
         idpProvider: OpenIdConnect,
         authority: issuerUrl,
         client_id: clientId,
-        scope:
-        'audience:server:client_id:kyma-client audience:server:client_id:console openid email profile groups',
+        scope: scope,
```

Comment:
if we remove the code from `init-params.js` then we can simply use `scope || 'openid email profile'` here